conda_smithy/lint_recipe.py

```diff
@@ -89,7 +106,7 @@ def lintify(meta, recipe_dir=None, conda_forge=False):
     # find the meta.yaml within it.
     meta_fname = os.path.join(recipe_dir or '', 'meta.yaml')
 
-    source_section = get_section(meta, 'source', lints)
+    sources_section = get_section(meta, 'source', lints)
```

Comment:
The `source section` is always returned as a list now, so I renamed the variable to `sources_section`